File: python/bubble.py
import random

ram = [random.randint(0, 1000) for i in range(100)]

for i in range(len(ram)):
    for j in range(len(ram)-1-i):
        if ram[j] > ram[j+1]:
            tem = ram[j]
            ram[j] = ram[j+1]
            ram[j+1] = tem

print(ram)

File: nodejs/bubble.js
if (require.main === module) {
  main()
}

function bubble(arr) {
  let change = true
  while (change) {
    change = false
    for (let i = 0; i < arr.length - 1; i++) {
      if (arr[i] > arr[i + 1]) {
        tem = arr[i]
        arr[i] = arr[i + 1]
        arr[i + 1] = tem
        change = true
      }
    }
  }
  return arr
}

function main() {
  ram = []
  for (let i = 0; i < 100; i++) {
    Math.random()
    ram[i] = Math.floor(Math.random() * 1000)
  }
  console.log(ram)
  console.log(bubble(ram))
}
